eng/versioning/version_client.txt

```diff
@@ -126,6 +126,7 @@ com.azure:azure-developer-devcenter;1.0.1;1.1.0-beta.1
 com.azure:azure-developer-loadtesting;1.0.14;1.1.0-beta.1
 com.azure:azure-digitaltwins-core;1.3.21;1.4.0-beta.1
 com.azure:azure-e2e;1.0.0-beta.1;1.0.0-beta.1
+com.azure:azure-health-deidentification;1.0.0;1.0.0-beta.1
```

Comment:
```suggestion
com.azure:azure-health-deidentification;1.0.0-beta.1;1.0.0-beta.1
```

Reply:
For the initial commit all our SDKs are using this version so I would keep it if that is not a problem